test/unit/lodash.spec.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+var Guy = require('./Guy');
+var chance = require('chance').Chance();
+
+describe('Lodash training', function ()
+{
+
+    describe('constant', function ()
+    {
+        it('should create a function that returns value.', function ()
+        {
+            var params = datasets.constant();
+            it('should check if first parameter is an object',function()
+            {
+                expect(params[0]).to.be.an('object');
+            });
+            it('should check objects values',function()
+            {
+                expect(params[0]).to.have.property('fred');
+                expect(params[0]).to.have.property('pebbles');
+                expect(params[0]).to.have.deep.property('john.age', 36);
+            });
+            it('should check if function returns an object', function (){
+                var getter = _.constant(params[0]);
+                expect(getter()).to.be.an('object');
+            });
+            it('should check if function returns an object', function (){
+
+                expect(_.constant.apply(_, params)).to.be.a('function');
+            });
+        });
+    });
+
+    describe('conforms', function ()
+    {
+        var params = datasets.constant();
+        it('should check if first parameter is an object',function()
+        {
+           expect(params[0]).to.be.an('object');
+        });
+        it('should check objects values',function()
+        {
+           expect(params[0]).to.have.property('fred');
+           expect(params[0]).to.have.property('pebbles');
+           expect(params[0]).to.have.deep.property('john.age', 36);
+        });
+        it('should check if function returns an object', function (){
+           var getter = _.constant(params[0]);
+           expect(getter()).to.be.an('object');
+        });
+        it('should create a function that returns value.', function ()
+        {
+            expect(_.constant.apply(_, params)).to.be.a('function');
+        });
+    });
+
+    describe.only('flow', function ()
+    {
+        var params = datasets.flow();
+        it('should check if first parameter is a function',function()
+        {
+           expect(params[0]).to.be.a('function');
+        });
+        it('should check if function return proper value', function (){
+           var addDivide = _.flow(_.divide, params[0]);
```

Comment:
```_.divide``` is not provided by lodash 3.10.1 which is defined in current ```package.json```. Test always fails.